src/schema/users.ts

```diff
@@ -1300,6 +1302,8 @@ export const resolvers: IResolvers<any, Context> = {
         flags,
       });
 
+      await resubscribeUser(cio, ctx.userId);
```

Comment:
We subscribe to everything or?

Reply:
cio remembers the last state, so i'm just removing the user from the global unsubscribe because they clearly want to receive email at this point